plugins/database/cassandra/connection_producer.go

```diff
@@ -125,15 +125,11 @@ func (c *cassandraConnectionProducer) Initialize(ctx context.Context, req dbplug
 
 	if req.VerifyConnection {
 		if _, err := c.Connection(ctx); err != nil {
-			return dbplugin.InitializeResponse{}, errwrap.Wrapf("error verifying connection: {{err}}", err)
+			return errwrap.Wrapf("error verifying connection: {{err}}", err)
```

Comment:
nit: could use this opportunity to change these to `fmt.Errorf("blah %w", err)` while we're here

Reply:
I had held off on this because we hadn't updated `errwrap` to 1.1 in a previous PR, but it looks like it was updated shortly after that PR was merged. I'll update accordingly and as I see them.